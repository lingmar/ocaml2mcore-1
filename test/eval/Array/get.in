let _ =
  let a = Array.init 5 (fun x -> x) in
  print_int (Array.get a 0);
  print_int (Array.get a 1);
  print_int (Array.get a 2);
  print_int (Array.get a 3);
  print_int (Array.get a 4);
  print_endline "";

  print_int (a.(0));
  print_int (a.(1));
  print_int (a.(2));
  print_int (a.(3));
  print_int (a.(4));
  print_endline "";

  let a = Array.init 4 (fun x -> 0.0) in
  print_float (Array.get a 0);
  print_float (Array.get a 1);
  print_float (Array.get a 2);
  print_float (Array.get a 3);
  print_endline "";

  print_float (a.(0));
  print_float (a.(1));
  print_float (a.(2));
  print_float (a.(3));
  print_endline "";

  ()
