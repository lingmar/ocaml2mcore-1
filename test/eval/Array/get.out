01234
01234
0.00.00.00.0
0.00.00.00.0